src/components/Input/Input.module.css

```diff
@@ -19,6 +19,5 @@
 
 .input-centered {
   width: 80%;
-  min-width: 280px;
```

Comment:
Can we document this change here in the PR history?

Reply:
This is just to adapt the input box on the Profile page to smaller mobile widths.